packages/taiko-client/bindings/metadata/metadata_ontake.go

```diff
@@ -0,0 +1,164 @@
+package metadata
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/taikoxyz/taiko-mono/packages/taiko-client/bindings"
+)
+
+// Ensure TaikoDataBlockMetadata2 implements TaikoBlockMetaData.
+var _ TaikoBlockMetaData = (*TaikoDataBlockMetadataOntake)(nil)
+
+// TaikoDataBlockMetadataOntake is the metadata of an ontake Taiko block.
+type TaikoDataBlockMetadataOntake struct {
+	bindings.TaikoDataBlockMetadataV2
+	types.Log
+}
+
+// NewTaikoDataBlockMetadata2 creates a new instance of TaikoDataBlockMetadata2
+// from the TaikoL1.BlockProposed2 event.
```

Comment:
https://github.com/taikoxyz/taiko-mono/pull/17734/commits/d39aca0eef0e4b8499b91f4bfa2aa55db274415b
The event has renamed

Reply:
Updated